bindings_test.go

```diff
@@ -109,7 +109,7 @@ func TestTopic(t *testing.T) {
 						t.Errorf("%s WAS called on %s with body %s", key, tc.request, body)
 					}
 					assert.Equal(t, tc.request, body, "correct request body")
-				case <-time.After(10 * time.Millisecond):
+				case <-time.After(10 * time.Second):
```

Comment:
Isn't this a bit much, feels like `1 * time.Second` would be enough?

Reply:
It probably is, unless the Travis instance is particularly slow.